services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy

```diff
@@ -0,0 +1,124 @@
+package org.openkilda.functionaltests.spec.northbound.switches
+
+import org.openkilda.functionaltests.BaseSpecification
+import org.openkilda.messaging.info.meter.SwitchMeterEntries
+import org.openkilda.testing.Constants
+import org.openkilda.testing.model.topology.TopologyDefinition
+import org.openkilda.testing.service.northbound.NorthboundService
+
+import org.junit.Ignore
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.client.HttpClientErrorException
+import spock.lang.Issue
+
+class MetersSpec extends BaseSpecification {
+
+    @Autowired
+    TopologyDefinition topology
+    @Autowired
+    NorthboundService northboundService
+
+    def "Unable to delete meter with invalid ID"() {
+        given: "A switch"
+        def sw = topology.getActiveSwitches().find { it.ofVersion == "OF_13" }
+
+        when: "Try to delete meter with invalid ID"
+        northboundService.deleteMeter(sw.getDpId(), -1)
```

Comment:
Let's extend test and try to delete meter with id=0 as well. So we should have two tests.

Reply:
done